src/Sulu/Bundle/LocationBundle/Content/Types/LocationContentType.php

```diff
@@ -137,4 +138,30 @@ private function getCountries()
 
         return $countries;
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function exportData($propertyValue)
+    {
+        if (is_string($propertyValue)) {
+            return $propertyValue;
+        }
+
+        return '';
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function importData(
+        NodeInterface $node,
+        PropertyInterface $property,
+        $userId,
+        $webspaceKey,
+        $languageCode,
+        $segmentKey = null
+    ) {
+        $this->write($node, $property, $userId, $webspaceKey, $languageCode, $segmentKey);
```

Comment:
could this be a "default" implementation? maybe in the `ComplexContentType`? or in a trait?